.github/workflows/releases.yml

```
@@ -3,7 +3,7 @@ name: Generate Release Notes
 on:
   push:
     tags:
-      - '[0-9]*.[0-9]*.[0-9]*' # Triggers the workflow when a tag matching the version pattern is pushed (e.g., 11.1.1)
+      - '*.*.*' # Triggers the workflow when a tag matching the version pattern is pushed (e.g., 11.1.1)
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)